Pipes - <Nkiru> - <Calculator.rb>

```diff
@@ -0,0 +1,103 @@
+# Nkiru Onwuneme
+# Last edited 8/11/17
+# Create a calculator
+
+##Issues:  user entry set to interger or in Method - fixed
+# consolidate methods
+
+# Methods for computing math operations
+def add(num_1, num_2)
+  result = num_1 + num_2
```

Comment:
I like that each of these is its own method. It makes it easy to find where in the code a certain operation occurs, and means that if you need to add a long complex operation (think finding the least common multiple of 2 numbers) the pattern will still work.